libjava/lib/src/main/java/com/mapbox/services/directions/v5/MapboxDirections.java

```diff
@@ -458,6 +444,11 @@ public Builder setClientAppName(String appName) {
       return this;
     }
 
+    public Builder setBaseUrl(String baseUrl) {
```

Comment:
Javadoc? Looks like we are also missing javadoc for the method above: `setClientAppName`